[2025-03-12T10:00:24.679+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hourly_processing_pipeline.transform_silver_hourly_incremental manual__2025-03-12T10:00:18.481853+00:00 [queued]>
[2025-03-12T10:00:24.684+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hourly_processing_pipeline.transform_silver_hourly_incremental manual__2025-03-12T10:00:18.481853+00:00 [queued]>
[2025-03-12T10:00:24.684+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-12T10:00:24.797+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): transform_silver_hourly_incremental> on 2025-03-12 10:00:18.481853+00:00
[2025-03-12T10:00:24.800+0000] {standard_task_runner.py:57} INFO - Started process 28485 to run task
[2025-03-12T10:00:24.802+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'hourly_processing_pipeline', 'transform_silver_hourly_incremental', 'manual__2025-03-12T10:00:18.481853+00:00', '--job-id', '2328', '--raw', '--subdir', 'DAGS_FOLDER/hourly_processing_pipeline.py', '--cfg-path', '/tmp/tmprcc6w2qb']
[2025-03-12T10:00:24.805+0000] {standard_task_runner.py:85} INFO - Job 2328: Subtask transform_silver_hourly_incremental
[2025-03-12T10:00:24.850+0000] {task_command.py:416} INFO - Running <TaskInstance: hourly_processing_pipeline.transform_silver_hourly_incremental manual__2025-03-12T10:00:18.481853+00:00 [running]> on host c8e526380ef3
[2025-03-12T10:00:24.912+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andrew Tran' AIRFLOW_CTX_DAG_ID='hourly_processing_pipeline' AIRFLOW_CTX_TASK_ID='transform_silver_hourly_incremental' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T10:00:18.481853+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T10:00:18.481853+00:00'
[2025-03-12T10:00:24.913+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-12T10:00:24.914+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /opt/***/dbt/playstudio_project && export WAREHOUSE_HOST=None && export WAREHOUSE_PASSWORD=None && export WAREHOUSE_USER=None && dbt run --select tag:silver_hourly_incremental --target prod']
[2025-03-12T10:00:24.921+0000] {subprocess.py:86} INFO - Output:
[2025-03-12T10:00:26.542+0000] {subprocess.py:93} INFO - [0m10:00:26  Running with dbt=1.7.18
[2025-03-12T10:00:26.651+0000] {subprocess.py:93} INFO - [0m10:00:26  Registered adapter: postgres=1.7.18
[2025-03-12T10:00:26.660+0000] {subprocess.py:93} INFO - [0m10:00:26  Unable to do partial parsing because of a version mismatch
[2025-03-12T10:00:27.179+0000] {subprocess.py:93} INFO - [0m10:00:27  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-12T10:00:27.180+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-12T10:00:27.180+0000] {subprocess.py:93} INFO - - models.playstudio_project.example
[2025-03-12T10:00:27.195+0000] {subprocess.py:93} INFO - [0m10:00:27  Found 3 models, 2 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-03-12T10:00:27.195+0000] {subprocess.py:93} INFO - [0m10:00:27
[2025-03-12T10:00:43.286+0000] {subprocess.py:93} INFO - [0m10:00:43
[2025-03-12T10:00:43.300+0000] {subprocess.py:93} INFO - [0m10:00:43  Finished running  in 0 hours 0 minutes and 16.08 seconds (16.08s).
[2025-03-12T10:00:43.300+0000] {subprocess.py:93} INFO - [0m10:00:43  Encountered an error:
[2025-03-12T10:00:43.300+0000] {subprocess.py:93} INFO - Database Error
[2025-03-12T10:00:43.300+0000] {subprocess.py:93} INFO -   could not translate host name "local" to address: Temporary failure in name resolution
[2025-03-12T10:00:43.301+0000] {subprocess.py:93} INFO - 
[2025-03-12T10:00:44.613+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-03-12T10:00:44.632+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-03-12T10:00:44.638+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=hourly_processing_pipeline, task_id=transform_silver_hourly_incremental, execution_date=20250312T100018, start_date=20250312T100024, end_date=20250312T100044
[2025-03-12T10:00:44.653+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 2328 for task transform_silver_hourly_incremental (Bash command failed. The command returned a non-zero exit code 2.; 28485)
[2025-03-12T10:00:44.682+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-03-12T10:00:44.733+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
